tests/messages/any/get.test.tsx

```diff
@@ -0,0 +1,197 @@
+
+        expect(res.messages.length).toBeGreaterThan(0);
+        res.messages.map((message) => {
+            expect(any.is.Post(message)).toStrictEqual(true);
+            if (any.is.Post(message)) expect(message.content.type).toStrictEqual(aimedType);
+        });
+    });
+
+    it("Try by tags", async () => {
+        const aimedTag = ["Test"];
+        const res = await any.Get({
+            tags: aimedTag,
+            pagination: 10,
+        });
+
+        expect(res.messages.length).toBeGreaterThan(0);
+        res.messages.map((message) => {
+            expect(any.is.Post(message)).toStrictEqual(true);
+            if (any.is.Post<{ tags: string }>(message)) expect(message.content.content?.tags).toStrictEqual(aimedTag);
+        });
+    });
+
+    it("Try by content Key", async () => {
+        const aimedKey = "InterPlanetaryCloud";
+        const res = await any.Get({
+            contentKeys: [aimedKey],
+            pagination: 10,
+        });
+
+        expect(res.messages.length).toBeGreaterThan(0);
+        res.messages.map((message) => {
+            expect(any.is.Aggregate(message)).toStrictEqual(true);
+            if (any.is.Aggregate(message)) expect(message.content.key).toStrictEqual(aimedKey);
+        });
+    });
+
+    it("Try by timestamp", async () => {
+        const aimedStartTime = new Date(1673882430814);
+        const aimedEndTime = new Date(1673882506494);
+        const res = await any.Get({
+            startDate: aimedStartTime,
+            endDate: aimedEndTime,
+            pagination: 5,
+        });
+
+        res.messages.map((item) => {
+            const date = new Date(item.time * 1000);
+            expect(date.valueOf()).toBeGreaterThanOrEqual(aimedStartTime.valueOf());
+            expect(date.valueOf()).toBeLessThanOrEqual(aimedEndTime.valueOf());
+        });
+    });
+
+    it("If a specific message does not exist, it should failed", async () => {
+        await expect(
+            any.Get({
+                hashes: ["w87e1e2ee2cbe88fa2923042b84b2f9c694w10005ca7dd40193838bf9bad18e12cw"],
+            }),
+        ).rejects.toThrow("No messages found");
+    });
+
+    it("try by all message type", async () => {
+        const typeArray = [
+            { type: MessageType.store, checker: any.is.Store },
+            { type: MessageType.post, checker: any.is.Post },
+            { type: MessageType.forget, checker: any.is.Forget },
+            { type: MessageType.aggregate, checker: any.is.Aggregate },
+            { type: MessageType.program, checker: any.is.Program },
+        ];
+
+        const checkTypeList = (messagesList: BaseMessage[], fctChecker: (message: BaseMessage) => boolean): boolean => {
+            let res = true;
+            messagesList.map((message) => {
+                if (fctChecker(message) === false) {
+                    console.error(`Message type failed on type: ${message.type}`);
+                    res = false;
+                }
+            });
```

Comment:
[Array.every](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) seems to be cleaner. The console.error won't be printed to stdout anyways.
```javascript
const checkTypeList = (messagesList: BaseMessage[], fctChecker: (message: BaseMessage) => boolean): boolean => ( messageList.every(fctChecker) )
```

Reply:
Updated made here: [af8c768](https://github.com/aleph-im/aleph-sdk-ts/pull/117/commits/af8c768713282208300900b3f44b1dbd034a2f59) thanks :D